ccpp/suites/suite_FV3_GFS_v15_thompson_mynn_RRTMGP.xml

```diff
@@ -0,0 +1,100 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<suite name="FV3_GFS_v15_thompson_mynn_RRTMGP" version="1">
```

Comment:
Do you also want to add a suite file for suite_FV3_GFS_v15_thompson_mynn_lam3km.xml? The suite_FV3_GFS_v15_thompson_mynn.xml is used for low resolution test (in RT), if you want to test it in high resolution LAM, you may need to add a new suite file for suite_FV3_GFS_v15_thompson_mynn_lam3km.xml

Reply:
We don't need to do this at this time.